### ISSUE TITLE
Obsidian Image Problems

### ISSUE THREAD

**Issue by LynnXie00** [theohbrothers/ConvertOneNote2MarkDown#150]
I actually don't know if I should put it as a bug or a feature. I'm using this tool to convert onenote to md for obsidian. Since the owner of this repository also recommended obsidian, I believe we are using it for the same reason...

## Bug

The images in the successfully converted md files look like this
```
![Machine generated alternative text: xxx ](media/单弦谱_Happy-birthday-image1.png){width="7.9375in" height="4.645833333333333in"}
```

But this is not the image format that Obsidian would accept. Obsidian accept image format as below, in which the numbers represent pixels. 
```
![Machine generated alternative text: xxx |793x464](media/单弦谱_Happy-birthday-image1.png)
```

## Expectation

Images are sized correctly.

## Discussion

So now the images look like this in Obsidian
[screenshot of Obsidian](https://i.ibb.co/pbJwzxC/Snipaste-2023-02-04-18-24-28.png)
## Environment

**Output of Configuration**

```powershell
Configuration:
dryRun: 0
notesdestpath: C:\Users\verac\OneDrive\Documents\Notebooks
targetNotebook:
usedocx: 2
keepdocx: 1
docxNamingConvention: 1
prefixFolders: 1
mdFileNameAndFolderNameMaxLength: 200
medialocation: 2
conversion: markdown-simple_tables-multiline_tables-grid_tables+pipe_tables
headerTimestampEnabled: 1
keepspaces: 1
keepescape: 1
newlineCharacter: 2
exportPdf: 1
```

**Output of `$PSVersionTable`**

```powershell
PS > $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.0.13
PSEdition                      Core
GitCommitId                    7.0.13
OS                             Microsoft Windows 10.0.22622
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by leojonathanoh** [comment]
You would need to adjust the pandoc extensions to get the `.md` flavor you want.

See: https://github.com/theohbrothers/ConvertOneNote2MarkDown/blob/c84686a66eecad4f002ecc5cc294ff0e8ef2b7c1/config.example.ps1#L52-L65

I don't use obsidian myself, so hopefully others may know the correct pandoc extensions to use.

**Comment by LynnXie00** [comment]
> https://pandoc.org/MANUAL.html#options

Thank you. I swear I will figure it out today. Hate to leave it tomorrow.

**Comment by leojonathanoh** [comment]
i feel your pain... i was there before about 1.5 years ago :) though i spent a good week or two working on script :P

**Comment by leojonathanoh** [comment]
Perhaps you can try this https://github.com/theohbrothers/ConvertOneNote2MarkDown/issues/145#issuecomment-1408720530

**Comment by leojonathanoh** [comment]
Just found #123, hope it helps!

**Comment by LynnXie00** [comment]
After trying for hours.... The best solution I found for Obsidian is:
```
$conversion = 'markdown_mmd-simple_tables-multiline_tables-grid_tables+pipe_tables-mmd_link_attributes'
```
This way, all the web clips, highlights, nested tables, will all be converted without any problems. However, the ```<img>``` tag cannot be displayed by obsidian if the note contains clipped web images. I found that[ this plugin](https://github.com/talengu/obsidian-local-img-plugin) can make``` <img> ```tag visible in Obsidian reading mode.

I hope my experimenting can help others.... lol....


Now, I'm going to try to solve the other issue by experimenting more....
